terraform/docs/applying-terraform.md

```diff
@@ -7,60 +7,49 @@ The EKS cluster is deployed via Terraform in two stages. See [adr-3] for backgro
 
 ## Prerequisites
 
-1. Some of the deployments below require external [tfvars
-files](../terraform/deployments/variables). To create a new environment, you
-will need to copy the variables directory for an existing environment (e.g.
-[integration](../deployments/variables/integration)) and modify as
-appropriate.
-    1. `cluster-infrastructure` deployment assumes that there is a Fastly CDN service
-       and requires a value (`www_dns_validation_rdata`) for creating the DNS validation of the Fastly domain. You can
-       either use a dummy value or look at setting up the [CDN service](../../docs/setting-up-content-delivery-network.md)
-1. Each of the deployments (except `ecr`) requires a [backend config
-file](https://www.terraform.io/docs/language/settings/backends/configuration.html#partial-configuration)
-You can use an existing backend file as a template.
-1. `cluster-services` deployment requires some [prerequisite secrets](../../docs/prerequisite-secrets.md)
+1. `cluster-infrastructure` deployment assumes that there is a Fastly CDN service
+   and requires a value (`www_dns_validation_rdata`) for creating the DNS validation of the Fastly domain. You can
+   either use a dummy value or look at setting up the [CDN service](../../docs/setting-up-content-delivery-network.md)
+2. `cluster-services` deployment requires some [prerequisite secrets](../../docs/prerequisite-secrets.md)
 which are not generated automatically. Create these secrets before running
-`terraform apply` for the first time.
+the Terraform apply for the first time.
 
 ## Deployment
 
-**We no longer have any deployment automation for Terraform** (since the demise of Big Concourse).
-
-For testing before merging to `main`, we can run Terraform locally against the test account.
+All terraform modules in this repository are now deployed via Terraform Cloud.
+To test changes before merging into main, open a PR and a plan will automatically
+start for your branch.
 
 When turning up from scratch, deploy the root modules in this order:
 
-1. [`terraform-lock`](../deployments/terraform-lock): creates the
-   DynamoDB table which Terraform uses to control concurrent access to the
-   state files in S3.
-1. [`ecr`](../deployments/ecr) (test and production accounts only): creates the ECR container registry from
+1. [`tfc-bootstrap`](../deployments/tfc-bootstrap): bootstraps Terraform Cloud and creates `tfc-configuration`
+workspace which is used to manage the other Terraform module workspaces.
```

Comment:
optional/trivial: passive voice doesn't add much here so prefer active (debatable rule of thumb but v common in tech writing style guides including [ours](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style))
```suggestion
workspace which manages the other Terraform module workspaces.
```